server/errors/errors.go

```diff
@@ -84,8 +84,8 @@ func RelationNotFound(relation string, typeName string, tuple *openfgapb.TupleKe
 	return status.Error(codes.Code(openfgapb.ErrorCode_relation_not_found), msg)
 }
 
-func EmptyRelationDefinition(typeDefinition, relation string) error {
-	return status.Error(codes.Code(openfgapb.ErrorCode_empty_relation_definition), fmt.Sprintf("Type definition '%s' contains an empty relation definition '%s'", typeDefinition, relation))
```

Comment:
This is a user-facing error. I feel like the existing message is clearer. https://auth0.slack.com/archives/C033LQD4S11/p1662068452184499?thread_ts=1662066590.039589&cid=C033LQD4S11

Reply:
How about now? Previously it was kind of wrong. Do you have any suggestions? It is in the case a user has something like
```
type doc
  relations
    define owner  // and nothing else
```

Reply:
My nitpick is that we don't mention the word `rewrite` anywhere in our documentation.

How about `the definition of relation R on type T is invalid` ?

Reply:
Oh yeah, that's much better. Thanks! Updated.